### PR TITLE
6.5.2

### PR DESCRIPTION
**Pull request by thomaslee**
Commits 3c5db2b and 4ca8941 are "noise" from the `gbp import-orig --uscan`. Everything else is me I think. Again, I'll need to push `pristine-tar`, `upstream` branches and an `upstream` tag once this is merged to keep gbp happy.

One thing that raised an eyebrow: `dh_auto_install` behavior with compat levels over 10. debhelper will stomp down `INSTALL="install --strip-program=true"`. This caused a problem during the install phase for spin because they modified the build to install the spin binary under `$(CURDIR)/debian/spin/bin`.

That directory doesn't exist at build time, `install -D` is not set by debhelper and there's no obvious way to override it (see [debhelper source](https://github.com/Debian/debhelper/blob/0d81dac9cf7b2be625b6265d55e48da1fe4eceb2/lib/Debian/Debhelper/Buildsystem/makefile.pm#L157-L159)). I also get a warning that `--strip-program=true` is ignored because `-s` is not specified. Something not quite right there, feel like I'm misunderstanding how it's supposed to fit together. Anyway, opted out of `dh_auto_install` all together (this was recommended by debhelper docs) and things work fine.

As I mentioned in email, omitted pre-compiled binaries in `Bin` during the repack along with the book errata for now. I'll raise both upstream.

There's a few other things that shook out of this release that I'd like to take up with upstream, like all the permissions issues with examples (see `override_dh_perms`).

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,13 @@
+spin (6.5.2+dfsg-1) UNRELEASED; urgency=medium
+
+  * New upstream release 6.5.2
+  * Bump Standards-Version to 4.5.0
+  * debian-compat 12
+  * Fix various build and lintian issues
+  * Exclude binaries and docs of questionable origin for now
```
Comment (tmancill) [inline debian/changelog]: Maybe clarify this statement a bit - at first I thought you were excluding binaries from the build - e.g.:

> "During repack of upstream tarball exclude binaries included in the upstream tarball. 
 Also exclude docs until the copyright can be determined"

**Review by tmancill — APPROVED** [review]
It looks good to me.  I suggested one tweak to the changelog wording that you should feel free to ignore.

File: debian/changelog (hunk 1 above)
Comment (thomaslee) [reply debian/changelog]: You're right, could be both clearer and less dismissive. Will get that updated.

Comment (thomaslee) [reply debian/changelog]: Done. I'll get this merged, but holler out if anything else needs work.

**Comment by thomaslee** [comment]
`upstream`, `pristine-tar` branches and `upstream/6.5.2+dfsg` tag pushed

**Pull request merged**